plat/kvm/x86/setup.c

```diff
@@ -287,6 +287,12 @@ static inline int cmdline_init(struct ukplat_bootinfo *bi)
 
 static void __noreturn _ukplat_entry2(void)
 {
+	/* It's not possible to unwind past this function, because the stack
+	 * pointer was overwritten in lcpu_arch_jump_to. Therefore, mark the
+	 * previous instruction pointer as undefined, so that debuggers or
+	 * profilers stop unwinding here.
+	 */
+	__asm__(".cfi_undefined rip\n");
```

Comment:
Maybe this could go as macro definition into the cfi header, too? I am convinced that we want to use the exact same statement also in the other platforms (Xen, etc.)